zerver/tests/test_messages.py

```diff
@@ -184,9 +184,6 @@ def create_user(self, email: Text) -> UserProfile:
         return get_user(email, get_realm(subdomain))
 
     @slow("Sends a large number of messages")
-    @override_settings(CROSS_REALM_BOT_EMAILS=['feedback@zulip.com',
-                                               'welcome-bot@zulip.com',
-                                               'support@3.example.com'])
```

Comment:
Why did you move this to be the `with` statement below?

Reply:
So that we can create support bot through `self.create_user`.  If we try to create this bot using modified settings, `self.create_user` will fail because we cannot create a system bot through normal registration flow.